base_report_to_printer/tests/test_printing_printer.py

```diff
@@ -55,7 +59,7 @@ def test_print_report(self, cups):
         with mock.patch('%s.mkstemp' % model) as mkstemp:
             mkstemp.return_value = fd, file_name
             printer = self.new_record()
-            printer.print_document('report_name', 'content to print', 'pdf')
+            printer.print_document(self.report, 'content to print', 'pdf')
```

Comment:
@Yajo you changed this before, isn't it? Do you see this correct?

Reply:
@Yajo, please confirm me this

Reply:
The test was "wrong" here.

The argument is already a `printing.printer` recordset everywhere in the code, examples:
https://github.com/OCA/report-print-send/blob/11.0/base_report_to_printer/models/ir_actions_report.py#L115
https://github.com/OCA/report-print-send/blob/11.0/base_report_to_printer/models/ir_actions_report.py#L146

Reply:
You're right, this test must remain untouched.